test/unit/test_schedule.cpp

```diff
@@ -323,42 +259,38 @@ TEST(schedule, poisson_offset) {
 
     const double offset = 3.3;
 
-    std::mt19937_64 G1;
-    G1.discard(300);
-
+    auto sched1 = poisson_schedule(.234*arb::units::kHz);
+    sched1.discard(300);
     std::vector<time_type> expected;
-    for (auto t: as_vector(poisson_schedule(.234, G1).events(0., 100.))) {
+    for (auto t: as_vector(sched1.events(0., 100.))) {
         t += offset;
         if (t<100.) {
             expected.push_back(t);
         }
     }
 
-    std::mt19937_64 G2;
-    G2.discard(300);
-
+    auto sched2 = poisson_schedule(offset*arb::units::ms, .234*arb::units::kHz);
+    sched2.discard(300);
     EXPECT_TRUE(seq_almost_eq<time_type>(expected,
-        as_vector(poisson_schedule(offset, .234, G2).events(0., 100.))));
+                                         as_vector(sched2.events(0., 100.))));
 }
 
 TEST(schedule, poisson_offset_reset) {
     SCOPED_TRACE("poisson_reset");
-    std::mt19937_64 G;
-    G.discard(400);
-    run_reset_check(poisson_schedule(3.3, 9.1, G), 1, 10, 7);
+    auto sched = poisson_schedule(3.3*arb::units::ms, 0.81*arb::units::kHz);
+    sched.discard(400);
+    run_reset_check(sched, 1, 10, 7);
 }
 
 TEST(schedule, poisson_tstop) {
     SCOPED_TRACE("poisson_tstop");
-    std::mt19937_64 G;
-    G.discard(500);
-
-    const double tstop = 50;
```

Comment:
why not reviving this variable instead of writing the literal `50` in 2 places?

Reply:
Sure.